packages/apps/job-launcher/server/src/modules/job/job.service.ts

```diff
@@ -261,18 +262,78 @@ export class JobService {
     if ((manifest as CvatManifestDto)?.annotation?.type) {
       await this.sendWebhook(
         this.configService.get<string>(
-          ConfigNames.EXCHANGE_ORACLE_WEBHOOK_URL,
+          ConfigNames.CVAT_EXCHANGE_ORACLE_WEBHOOK_URL,
         )!,
         {
           escrowAddress: jobEntity.escrowAddress,
           chainId: jobEntity.chainId,
+          eventType: EventType.ESCROW_CREATED
         },
       );
     }
 
     return jobEntity;
   }
 
+  public async requestToCancelJob(
+    userId: number,
+    id: number,
+  ): Promise<boolean> {
+    const jobEntity = await this.jobRepository.findOne({ id, userId });
```

Comment:
We need to check that the Escrow is in a status that allows to cancel it:

- balance >0
- status != Complete
- status != Paid